aderyn_core/src/ast/ast_nodes.rs

```diff
@@ -639,7 +639,7 @@ ast_node!(
 ast_node!(
     #[derive(Hash)]
     struct Return {
-        function_return_parameters: NodeID,
+        function_return_parameters: Option<NodeID>,
```

Comment:
This should have been optional. Because  it wasn't, it has affect at least the contracts where modifier has a return statement.

FIX